aiidalab/metadata.py

```diff
@@ -64,7 +64,12 @@ def _parse_setup_cfg(setup_cfg):
         "state", _map_development_state(metadata_pep426.get("classifiers", []))
     )
 
-    yield "categories", aiidalab.get("categories", [])
+    # Allow passing single category and convert to list
+    # and allow parse line seprated string as list
+    categories = aiidalab.get("categories", [])
+    if isinstance(categories, str):
```

Comment:
Could you show how does a "line separated string" look like in practice in the config file? Is that what was supported before this change?

Here's what I have in mine setup.cfg

```
categories =
    quantum chemistry
    UV/VIS spectroscopy
    atmospheric chemistry
```

Reply:
I think the categories need to go into the subset of list "quantum" etc. but cannot be something else. I'll try to fined where it is defined.

Reply:
That's defined at https://github.com/aiidalab/aiidalab-registry/blob/master/categories.yaml and read by https://github.com/aiidalab/aiidalab/blob/4393c31530bb0062b4dc4c20f5ae9f0ec75d3a7d/aiidalab/__main__.py#L619-L625

Reply:
Ah, if there are pre-defined categories then we need to add some validation.

Reply:
I think yes, maybe there is already checks for that. Your app is not registered yet correct? Maybe register it and we have a test with your app ;)

Reply:
I give it a quick test, there is no schema validation in terms of it but when building the gh-page it will failed for sure.

Reply:
Okay.

Yeah, I didn't register, I want to do that once I have a stable version (which includes the migration to AiiDA 2.0).